src/plugins/dashboard/public/application/dashboard_empty_screen.scss

```diff
@@ -1,9 +1,3 @@
-.dshAppContainer {
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-}
-
 .dshStartScreen {
```

Comment:
could combine the scss files together. so could remove this one and have single app.scss

Reply:
I think i prefer to have separate style sheets for each component? It helps me to see the structure better imo.

Also in the visualize plugin, src/plugins/visualize/public/application/components, the scss files are separated for each component as well